chart/cron-connector/README.md

```diff
@@ -0,0 +1,29 @@
+# Using helm chart for deployment
```

Comment:
Can you add the instructions for using it from the chart repository first please?